src/app/shared/components/grouped-list/grouped-list.component.html

```diff
@@ -1,11 +1,12 @@
 <ng-container *ngFor="let child of tree">
   <div class="grouped-section" *ngIf="child.name; else leaf">
-    <h4>{{ child.name | translate }}</h4>
+    <h4 [ngClass]="{ 'no-padding': isTable }">{{ child.name | translate }}</h4>
```

Comment:
please do not add this. Let's remove large padding for mat-cell inside cs-security-group-rule